webpack.config.js

```diff
@@ -25,7 +25,7 @@ module.exports  = {
         'ol-ext': path.resolve('./node_modules/ol-ext'),
         'bootstrap': path.resolve('./node_modules/bootstrap'),
         'angular-ui-bootstrap': path.resolve('./node_modules/angular-ui-bootstrap'),
-        'jquery': path.resolve('./node_modules/jquery')
+        'jquery': path.resolve('./node_modules/jquery/src/jquery')
```

Comment:
is this intended for the production build?

Reply:
I haven't test this in a production build, but  i believe it should be the same. The issue was that jQuery was not found using the previous path leading to `$ is not defined` errors

Reply:
allright :+1: